src/ameba/ast/variabling/variable.cr

```diff
@@ -174,7 +174,7 @@ module Ameba::AST
         node.accept self
       end
 
-      # @[AlwaysInline]
+      @[AlwaysInline]
```

Comment:
i have probably missed something, but could you please explain why do we need to ask the compiler to inline this method?

Reply:
IMO the method body is simple enough to inline it for a small perf improvement.